draft-ietf-tls-tls13.md

```diff
@@ -1619,6 +1603,7 @@ reducing handshake latency, as shown below.
          + ClientKeyShare
          + KnownConfiguration
          + EarlyDataIndication
+       (EncryptedExtensions)
```

Comment:
Personally, I'd rather have EncryptedExtensions mandatory, and simply have it empty if there are none.  Fewer chances for errors that way.